torch/_subclasses/meta_utils.py

```diff
@@ -845,6 +845,14 @@ def _empty_create_subclass(
                     )
                     inner_tensors[attr] = new_empty_tensor
 
+                if t.is_nested:
```

Comment:
At some point we should add some testing for `DTensor(NJT)` 😛 (the real test that "fakeifying inner tensors recursively" is working)

Reply:
Yes definitely, I didn't think too hard about nesting while writing all of this lol